lnwallet/transactions_test.go

```diff
@@ -251,6 +303,18 @@ func addTestHtlcs(t *testing.T, remote,
 func testVectors(t *testing.T, chanType channeldb.ChannelType, test testCase) {
 	tc := newTestContext(t)
 
+	// Determine which htlc set to use.
+	testHtlcs := testHtlcsSet1
+	if strings.Contains(test.Name, "same amount and preimage") {
```

Comment:
🧐

Reply:
yeah - weird I know 🙃 but the spec does actually define this test like this 🙈